src/Application/Messenger/Handler/SendRequestConsumer.php

```diff
@@ -56,27 +56,36 @@ public function __invoke(SendRequest $sendRequest): void
             $this->fail($sendRequest->id, "Validation failed: {$this->validator->getReason()}");
         }
 
-        $bodyRenderedHtml = $this->twig->render("{$sendRequest->templateKey}.html.twig", $sendRequest->params);
+        // Instantiate a new Swift Message object
+        $email = new \Swift_Message();
+
+        $pathToImages = __DIR__ . '/../../../../images/';
```

Comment:
This is a bit hard to follow. You can give `dirname()` a number of directory hops now, so doing this relative to `__DIR__` is now common.